go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/llm-d/llm-d-inference-scheduler
 
-go 1.24.1
-
-toolchain go1.24.2
+go 1.24.8
```

Comment:
Is there a reason why you upgraded GO here?

Reply:
Not needed - This update happens because of my local go version is 1.24.8. Please ignore any changes unrelated to metrics implementation.